Translations update from Servarr Weblate

ServarrAdmin opened the pull request:
Translations update from [Servarr Weblate](https://translate.servarr.com) for [Servarr/Whisparr](https://translate.servarr.com/projects/servarr/whisparr/).



Current translation status:

![Weblate translation status](https://translate.servarr.com/widget/servarr/whisparr/horizontal-auto.svg)

plz12345 reviewed (approved):
These are all non-EN changes, and we haven't really done any labels for non-EN.  Looks good for a clean merge.

weblate committed:
Multiple Translations updated by Weblate

ignore-downstream

Co-authored-by: 1 <1228553526@qq.com>
Co-authored-by: Alexander Balya <alexander.balya@gmail.com>
Co-authored-by: Ano10 <Ano10@users.noreply.translate.servarr.com>
Co-authored-by: Anonymous <noreply@weblate.org>
Co-authored-by: ArLab1 <arnaud.laberge@hotmail.com>
Co-authored-by: Ardenet <1213193613@qq.com>
Co-authored-by: Fixer <ygj59783@zslsz.com>
Co-authored-by: Florian Savouré <florian.savoure@gmail.com>
Co-authored-by: Gallyam Biktashev <gallyamb@gmail.com>
Co-authored-by: GkhnGRBZ <gkhn.gurbuz@hotmail.com>
Co-authored-by: HUi <huynguyeexn@gmail.com>
Co-authored-by: Havok Dan <havokdan@yahoo.com.br>
Co-authored-by: Hugoren Martinako <aumpfbahn@gmail.com>
Co-authored-by: Ilbebino <tommasobellandi08@gmail.com>
Co-authored-by: João Victor <joao.rodrigues@dcomp.ufs.br>
Co-authored-by: Kuzmich <kuzmich55@gmail.com>
Co-authored-by: Languages add-on <noreply-addon-languages@weblate.org>
Co-authored-by: Lars <lars.erik.heloe@gmail.com>
Co-authored-by: Lizandra Candido da Silva <lizandra.c.s@gmail.com>
Co-authored-by: Mailme Dashite <mailmedashite@protonmail.com>
Co-authored-by: Matti Meikäläinen <diefor-93@hotmail.com>
Co-authored-by: Michaa85 <michael.seipel@gmx.de>
Co-authored-by: Mizuyoru_TW <mizuyoru.tw@gmail.com>
Co-authored-by: NanderTGA <nander.roobaert@gmail.com>
Co-authored-by: Oskari Lavinto <olavinto@protonmail.com>
Co-authored-by: Robin Dadswell <robin@robindadswell.tech>
Co-authored-by: Rodion <rodyon009@gmail.com>
Co-authored-by: Tomer Horowitz <tomerh2001@gmail.com>
Co-authored-by: Tommy Au <smarttommyau@gmail.com>
Co-authored-by: Weblate <noreply@weblate.org>
Co-authored-by: farebyting <farelbyting@gmail.com>
Co-authored-by: fordas <fordas15@gmail.com>
Co-authored-by: hhjuhl <hans@kopula.dk>
Co-authored-by: hibomb <hibomb@hotmail.com>
Co-authored-by: keysuck <joshkkim@gmail.com>
Co-authored-by: marapavelka <mara.pavelka@gmail.com>
Co-authored-by: mryx007 <mryx@mail.de>
Co-authored-by: mytelegrambot <lacsonluxur@gmail.com>
Co-authored-by: qazwsxedc1d <13211175387@163.com>
Co-authored-by: warkurre86 <tom.novo.86@gmail.com>
Translate-URL: http://translate.servarr.com/projects/servarr/whisparr/tr/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/ar/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/bg/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/ca/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/cs/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/da/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/de/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/el/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/es/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/fa/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/fi/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/fr/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/he/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/hi/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/hr/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/hu/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/id/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/is/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/it/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/ja/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/ko/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/lv/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/nb_NO/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/nl/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/pl/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/pt/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/pt_BR/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/ro/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/ru/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/sk/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/sv/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/th/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/tr/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/uk/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/vi/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/zh_CN/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/zh_HANS/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/zh_Hans/
Translate-URL: https://translate.servarr.com/projects/servarr/whisparr/zh_TW/
Translation: Servarr/Whisparr
